TreeTracker/UI/Views/AddTree/AddTreeViewController.swift

```diff
@@ -191,3 +233,76 @@ private extension AddTreeViewModel.GPSAccuracy {
         }
     }
 }
+ // MARK: - AddTreeModel AVCapture Extension
```

Comment:
[suggestion:](https://conventionalcomments.org/#labels) AddTreeViewController raather than AddTreeModel?
```suggestion
 // MARK: - AddTreeViewController AVCapture Extension
```